docs/source/user/nalu_run/nalu_inp.rst

```diff
@@ -1139,6 +1139,275 @@ Time-step Control Options
+   additional information, which is helpful when troubleshooting a
+   simulation.
+
+.. inpfile:: sim_start
+
+   String indicating how OpenFAST should start.  For FSI
+   simulations, we suggest that the user set this to
+   ``trueRestart`` (no quotes).
+
+.. inpfile:: t_start
+
+   The start time of the FSI simulation.  This should be an
+   integer multiple of ``dt_FAST``, and should match ``t_end``
+   in the OpenFAST driver file.  Note that OpenFAST and
+   Nalu-Wind record time differently, so this variable will
+   not necessarily fit with the parameters in the
+   ``Time_Integrators`` section.
+
+.. inpfile:: t_max
+
+   The FSI simulation will end if the simulation time exceeds
+   this value.
+
+.. inpfile:: n_checkpoint
+
+   An integer indicating the frequency with which checkpoint files
+   will be written.  That is, a checkpoint will be written every
+   ``n_checkpoint`` timesteps.  Commonly this is set to correspond
+   to the number of steps in one rotor revolution.
+
+.. inpfile:: dt_FAST
+
+   The timestep used by OpenFAST for the FSI simulation.  This should
+   match ``DT`` in the OpenFAST input file (with extension .fst).
+   Commonly, this is 1/4 of the driver/Nalu/AMR-Wind timestep.
+
+.. inpfile:: turbine_name
+
+   This subsection includes parameters for a particular turbine.  Any
+   name may be provided by the user.  In the above example, the name
+   ``Turbine0`` was provided.  In the following, any variable appearing
+   in this subsection will be denoted ``turbine_name.variable_name`` for
+   clarity, but only the variable name should be included in the input
+   file.
+
+.. inpfile:: turbine_name.turbine_base_pos
+
+   Real vector indicating the location of the base of the tower.
+   This should match the variable with the same name in the OpenFAST
+   driver input file (with extension .yaml).
+
+.. inpfile:: turbine_name.turbine_hub_pos
+
+   Real vector indicating the location of the hub.
+   This should match the variable with the same name in the OpenFAST
+   driver input file (with extension .yaml).  Note that any height
+   change due to rotor tilt should be included in this variable, but
+   the effect of yaw should not be included.
+
+.. inpfile:: turbine_name.restart_filename
+
+   String (in double quotes) indicating the OpenFAST checkpoint file
+   that the FSI simulation will start from.  Do not include the
+   extension (.chkp).  This variable will have the format
+   ``"name.integer"``, where the ``integer`` is the number of steps taken
+   in the OpenFAST standalone simulation.  This should be equal to
+   ``t_end`` divided by ``dt_FAST`` from the OpenFAST standalone run.
+
+.. inpfile:: turbine_name.sim_type
+
+   String (in double quotes) indicating the type of OpenFAST simulation.
+   For FSI simulations, this should always be set to ``"ext-loads"``.
+
+.. inpfile:: turbine_name.blade_parts
+
+   List of strings (in double quotes) indicating the mesh element blocks
```

Comment:
Double quotes are not required for yaml to recognize strings.  Raw characters are fine.